db/migrate/20150622073727_add_redirects_for_topics_namespace.rb

```diff
@@ -0,0 +1,52 @@
+class AddRedirectsForTopicsNamespace < ActiveRecord::Migration
+  def up
+    Topic.published.only_parents.each do |topic|
+      original_topic_base_path = original_topic_base_path(topic)
```

Comment:
I think using the same name for the method and the local variable here makes things a little unclear.  It forces me to think about ruby's scoping rules when reading the code below.